website/source/index.html.erb

```diff
@@ -65,130 +65,75 @@ description: |-
     </div>
   </section>
 
-  <section id='static-dynamic' class='g-section bg-light'>
+  <section id='use-cases' class='g-section bg-light'>
     <div class='g-container'>
       <div class='intro'>
-        <h2>Service-based networking for dynamic infrastructure</h2>
-        <p>The shift from static infrastructure to dynamic
-          infrastructure changes the approach to networking from host-based to
-          service-based. Connectivity moves from the use of static IPs to
-          dynamic service discovery, and security moves from static firewalls to
-          service identity.</p>
-      </div>
-      <div class='g-timeline'>
-        <div>
-          <span class='line'></span>
-          <span class='line'>
-            <svg xmlns='http://www.w3.org/2000/svg' width='11' height='15' viewBox='0 0 11 15'>
-                <path fill='#CA2171' d='M0 0v15l5.499-3.751L11 7.5 5.499 3.749.002 0z'/>
-            </svg>
-          </span>
-          <span class='dot'></span>
-          <h3>Static</h3>
-          <span class='sub-heading'>Host-based networking</span>
-          <img src='/assets/images/consul-connect/svgs/static.svg' alt='static, host-based networking' class='static-callout' />
-        </div>
-        <div>
-          <span class='dot'></span>
-          <h3>Dynamic</h3>
-          <span class='sub-heading'>Service-based networking</span>
-          <div id='index-dynamic-animation'>
-            <%= inline_svg 'consul-connect/svgs/dynamic.svg' %>
-          </div>
-        </div>
-      </div>
-    </div>
-  </section>
-
-  <section id='use-cases' class='g-section'>
-    <div class='g-container'>
-      <div class='intro'>
-        <h2>Use Cases</h2>
+        <h2>Common Tasks</h2>
```

Comment:
thoughts on "Tasks Consul can do" or "What Consul can do"

Reply:
Maybe "What can you do with Consul?"

Reply:
Could we phrase this more as a call to action, something like "Put Consul to work?"